simulation_code/simulations.jl

```diff
@@ -228,8 +248,11 @@ end
         mu_Uit = mu_Vit + beta * cur_V
         # make choice
         ce_Uit = certainty_equivalent(alpha, mu_Uit, Sigma_Vit) # γ: uncertainty aversion
+
         c_it = nothing
-        if rand() < epsilon
+        if use_thompson
+            c_it = thompson_sampling(mu_V_i, Sigma_V_i, N)
```

Comment:
so what we should get returned here is the index of the item to be cnosumed, not the draw

Reply:
changed `c_it` to be 
```
c_it = argmax(draws)
```
in b089f70